zstacklib/zstacklib/utils/pci.py

```
@@ -38,9 +40,256 @@ def update_cache_devices(devices_dict):
+    try:
+        r, o, _ = bash_roe("hy-smi info")
+        if r == 0 and o:
+            # Normalize PCI address for comparison
+            normalized_pci = pci_address.lower()
+            # Check if PCI address is in output (handle various formats)
+            output_lower = o.lower()
+            # Try exact match first, then substring match
+            if normalized_pci in output_lower:
+                return True
+    except Exception:
+        pass
```

Ruff (0.14.14)

`try`-`except`-`pass` detected, consider logging the exception

(S110)